infra/helper.py

```diff
@@ -713,6 +715,29 @@ def build_fuzzers_impl(  # pylint: disable=too-many-arguments,too-many-locals,to
 
 def build_fuzzers(args):
   """Builds fuzzers."""
+  if args.engine == 'centipede':
+    # Centipede always requires separate binaries for sanitizers.
+    # An unsanitized binary, which Centipede requires for fuzzing.
+    unsanitized_bin_status = build_fuzzers_impl(args.project,
```

Comment:
nit: we can avoid a lot of duplication if we just do:


```python
for sanitizer in ('none', args.sanitizer):
  status = build_fuzzers_impl(...)
  if not status:
    return status
```

I reckon we can just early exit as well if any of these fail.

Reply:
Done, I made a little modification as the sanitized binary will be placed in a child directory, i.e.:
project
|-- binary
|-- project_address
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| -- binary